src/table/__tests__/TableWrapper.spec.jsx

```diff
@@ -115,32 +94,8 @@ describe('<TableWrapper />', () => {
     expect(handleKeyPress.handleTableWrapperKeyDown).to.have.been.calledOnce;
   });
 
-  it('should call setPageInfo when clicking next page and previous page button', async () => {
-    const { findByTitle, findByText } = render(
-      <TableWrapper
-        tableData={tableData}
-        setPageInfo={setPageInfo}
-        constraints={constraints}
-        selectionsAPI={selectionsAPI}
-        rootElement={rootElement}
-        keyboard={keyboard}
-        translator={translator}
-        rect={rect}
-      />
-    );
-    fireEvent.click(await findByTitle('SNTable.Pagination.NextPage'));
-
-    expect(setPageInfo).to.have.been.calledWith({ top: rowsPerPage, height: rowsPerPage });
-    expect(await findByText(`SNTable.Pagination.DisplayedRowsLabel`)).to.be.visible;
-
-    fireEvent.click(await findByTitle('SNTable.Pagination.PreviousPage'));
-
-    expect(setPageInfo).to.have.been.calledWith({ top: 0, height: rowsPerPage });
-    expect(await findByText(`SNTable.Pagination.DisplayedRowsLabel`)).to.be.visible;
-  });
-
-  it('should call setPageInfo when clicking last page and first page button', async () => {
```

Comment:
+1

Reply:
They are removed from here, since I test them in `TablePaginationActions.spec.jsx`. In there I split them up from 2 to for tests. There is no good way of waiting for it to rerender, so I run them seperately